cmd/cli/comm_trans.go

```diff
@@ -613,3 +613,54 @@ func printTx(tx *pb.Transaction) error {
 
 	return nil
 }
+
+func (c *CommTrans) GenTxInputsWithMergeUTXO(ctx context.Context) ([]*pb.TxInput, *pb.TxOutput, error) {
+	var fromAddr string
+	var err error
+	if c.From != "" {
+		fromAddr = c.From
+	} else {
+		fromAddr, err = readAddress(c.Keys)
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+
+	utxoInput := &pb.UtxoInput{
+		Bcname:   c.ChainName,
+		Address:  fromAddr,
+		NeedLock: false,
```

Comment:
Do not need lock UTXO? any change makes utxo conflict in a race condition?

Reply:
Thank you for your reminder.